tests/integration/local/test_dataframe_snapshot.py

```diff
@@ -0,0 +1,40 @@
+import PyRDF
+import unittest
+import os
+
+
+class RDataFrameSnapshot(unittest.TestCase):
+    """Test `Snapshot` functionality for RDataFrame"""
+
+    def test_snapshot_nrows(self):
+        """Test support for `Snapshot` in local backend"""
+        def fill_tree(treeName, fileName):
+            rdf = PyRDF.RDataFrame(100)
+            return rdf.Define("b1", "(int) rdfentry_")\
```

Comment:
Are the `(int)` and `(float)` castings really required?

Reply:
Not sure, probably not. I just took the function from the RDataFrame tutorials